doc/contributing/coding-conventions.chapter.md

```diff
@@ -500,7 +495,7 @@ Let's say Man-in-the-Middle (MITM) sits close to your network. Then instead of f
 
 - `https://` URLs are secure in methods 1, 2, 3;
 
-- `https://` URLs are not secure in method 5. When obtaining hashes with fake hash method, TLS checks are disabled. So refetch source hash from several different networks to exclude MITM scenario. Alternatively, use fake hash method to make Nix error, but instead of extracting hash from error, extract `https://` URL and prefetch it with method 1.
+- `https://` URLs are secure in method 5 *if* you use `""`, `lib.fakeHash`, `lib.fakeSha256`, or `lib.fakeSha512`.  If you use any other hash, `fetchurl` will pass `--insecure` to `curl` in order to be robust in the face of TLS certificate expiration and to allow fetching over protocols like HTTP which do not incur the overhead of TLS.
```

Comment:
Let's emphasize this exception. Also refer to single source of truth, otherwise we will have to maintain that list twice.

Does the shorter wording still reflect what you wanted to say?

```suggestion
- `https://` URLs are secure in method 5 *only if* you use one of the listed fake hashes. If you use any other hash, `fetchurl` will pass `--insecure` to `curl` and may then degrade to HTTP in case of TLS certificate expiration.
```

Reply:
> Does the shorter wording still reflect what you wanted to say?

Yes, except for the assumption that this only affects HTTP (I referenced HTTP as an example).  I'll apply this and push another commit.